README.md

```diff
@@ -1,13 +1,14 @@
 # lintr
-[![R build status](https://github.com/r-lib/lintr/workflows/R-CMD-check/badge.svg)](https://github.com/r-lib/lintr/actions)
+
+[![R build status](https://github.com/r-lib/lintr/workflows/R-CMD-check/badge.svg)](https://github.com/r-lib/lintr/actions/)
```

Comment:
Why is that?

Reply:
All Markdown elements should be separated by blank lines.

cf. https://yihui.org/en/2021/06/markdown-breath/

Reply:
And the trailing slash in the URL?

Reply:
Force of habit. I have reverted these changes. Although they won't cause any issues, the trailing slashes are unnecessary here.